### PR TITLE
Ux 568 improve sortable

### PR DESCRIPTION
**Pull request by jamiek-acl**
### 🛠 Purpose

https://aclgrc.atlassian.net/browse/UX-568
Did not do the "Nice to have" or "Stretch" items.

Added hasZebraStripes prop:
Screen Shot 2019-10-08 at 11 07 03 AM

Fixed overflow bug. Before:
Screen Shot 2019-10-07 at 4 06 51 PM

After:
Screen Shot 2019-10-07 at 4 06 36 PM

Changed `x` icon to a trashcan icon.
Screen Shot 2019-10-08 at 11 14 58 AM

### CODE REVIEW

File: packages/Sortable/src/Sortable.js [hunk 1]
```diff
@@ -60,13 +62,16 @@ const Sortable = ({ children, onChange, hasNumbers, onRemove, ...moreProps }) =>
     }
   };
 
+  console.log("hasZebraStripes", hasZebraStripes);
```
Comment (oscarkwan) [inline packages/Sortable/src/Sortable.js]: ✂️

Comment (jamiek-acl) [reply packages/Sortable/src/Sortable.js]: whoops

File: packages/Sortable/src/components/SortableItem/SortableItem.js [hunk 2]
```diff
@@ -50,12 +53,9 @@ const SortableItem = ({ children, index, hasNumbers, onRemove }) => {
           </div>
           {onRemove && (
             <div css={itemCloseStyles} data-pka-anchor="sortable.item.remove">
-              <Button.Close
-                a11yText={I18n.t("sortable.aria_remove")}
```
Comment (alexzherdev) [inline packages/Sortable/src/components/SortableItem/SortableItem.js]: Is it save to remove `sortable.aria_remove` string? And is there an announcement on the icon now?

Comment (jamiek-acl) [reply packages/Sortable/src/components/SortableItem/SortableItem.js]: Good catch

**Pull request merged**